extensions/HuggingFace/python/src/aiconfig_extension_hugging_face/remote_inference_client/conversational.py

```diff
@@ -33,9 +36,7 @@ def refine_completion_params(model_settings: dict[Any, Any]) -> dict[str, Any]:
     Doc Ref: https://huggingface.co/docs/huggingface_hub/package_reference/inference_client#huggingface_hub.InferenceClient.conversational
```

Comment:
TODO: Put a TODO for adding conversation history support (just focus on backend since UI is already similar to what Ryan did in https://github.com/lastmile-ai/aiconfig/pull/1209)